docs/index.rst

```diff
@@ -10,19 +13,19 @@ OpenQL is a framework for high-level quantum programming in C++/Python. The fram
 
    installation
    start
-   changelog
-   contributors
 
 .. toctree::
    :maxdepth: 3
    :caption: OpenQL Basics:
 
-   platform
    quantum_gate
-   classical_instructions
-   kernel
    program
+   kernel
+   classical_instructions
+   platform
    compiler_passes
+   changelog
```

Comment:
Adding a `..toctree::` header here would be nice as it separates the changelog and contributions visually from the rest of the documentation.